doc/requirements.txt

```diff
@@ -13,5 +14,7 @@ sphinx-copybutton
 sphinx-automodapi
 sphinx-gallery
 sphinxcontrib-bibtex==0.4.2
+tensorflow-tensorboard>=0.1.8
+tensorflow==1.3
```

Comment:
I feel really confused now :sweat_smile:

Reply:
This needs to be changed, good eye!